### PR TITLE
Hide  Lives (Header Right) when adaptive content

### PR DESCRIPTION
**Pull request by alanlanglois**
## Detailed purpose of the PR

Hide Header Right containing lives when content is Adaptive. 
https://trello.com/c/SKEsHgUA/1679-do-not-display-lives-and-counter-everywhere-when-module-or-chapter-isconditional
before: 
image
after: 
image

### Testing strategy

- [ ] I ensured my code is tested: 
 - checked on iOS Simulator
- [ ] I took care to mention breaking changes and extra libs if appropriate

### CODE REVIEW

File: src/containers/header-slide-right.test.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
+// @flow
+import renderer from 'react-test-renderer';
+import * as React from 'react';
+import {Component as HeaderSlideRight} from './header-slide-right';
+
+describe('header-slide-right', () => {
```
Comment (VincentCATILLON) [inline src/containers/header-slide-right.test.js:6]: HeaderSlideRight please :)

File: src/containers/header-slide-right.test.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        isGodModeEnabled={isGodModeEnabled}
+        isFastSlideEnabled={isFastSlideEnabled}
+        isAdaptive={isAdaptive}
+        toggleGodMode={jest.fn()}
+        toggleFastSlide={jest.fn()}
+      />
+    );
+
+    const headerSlideRightComponent = component.root.find(el => el.props.onPress);
+    expect(headerSlideRightComponent.props).toBeDefined();
```
Comment (VincentCATILLON) [inline src/containers/header-slide-right.test.js:27]: i dont understand what this test is supposed to test ?

Comment (alanlanglois) [reply src/containers/header-slide-right.test.js:27]: it checks the component HeaderSlideRight exists and has been rendered

Comment (VincentCATILLON) [reply src/containers/header-slide-right.test.js:27]: Maybe checking that props are defined is confusing for someone reading this test.
imho you should only test that headerSlideRightComponent is defined :)

Comment (alanlanglois) [reply src/containers/header-slide-right.test.js:27]: This way? expect(headerSlideRightComponent).toBeDefined();

**Comment by Fausto95** [comment]
I just figured out that the current implementation might no be the right way to go. That said, we might want to have the property `livesDisabled` set to false(in the engine configuration) whenever we want to consume an adaptive content.

**Comment by alanlanglois** [comment]
I'm not sure I get it?

**Comment by Fausto95** [comment]
Anyway, we still got the flash

**Comment by Fausto95** [comment]
Fixed by: https://github.com/CoorpAcademy/mobile/pull/524

**Pull request closed without merging**